boards/arm/96b_carbon_nrf51/doc/96b_carbon_nrf51.rst

```diff
@@ -0,0 +1,113 @@
+.. _96b_carbon_nrf51_board:
+
+96Boards Carbon nRF51
+#####################
+
+.. note::
+
+   If you're looking to reprogram the main STMicro part, see
+   :ref:`96b_carbon_board`. Users should not use this configuration
+   unless they want to reprogram the secondary chip which provides
+   Bluetooth connectivity.
+
+Overview
+********
+
+Zephyr applications use the 96b_carbon_nrf51 configuration to run on
+the secondary nRF51822 chip on the 96Boards Carbon. This chip provides
+Bluetooth functionality to the main STM32F401RET chip via SPI.
+
+See :ref:`96b_carbon_board` for a general overview of the board.
```

Comment:
can you please follow the template in doc/templates/board.tmpl to keep board documentation consistent? Otherwise looks good